examples/chat/client/chat.cc

```diff
@@ -273,16 +269,12 @@ int main(int argc, char** argv) {
   if (!absl::Base64Unescape(absl::GetFlag(FLAGS_room_id), &room_id)) {
     LOG(QFATAL) << "Failed to parse --room_id as base 64";
   }
-  const std::string& room_name = absl::GetFlag(FLAGS_room_name);
   std::unique_ptr<Room> room;
-  if (!room_name.empty()) {
-    room = absl::make_unique<Room>(stub.get(), room_name);
+  if (room_id.empty()) {
+    room = absl::make_unique<Room>(stub.get());
     room_id = room->Id();
     LOG(INFO) << "Join this room with --app_address=" << addr
               << " --room_id=" << absl::Base64Escape(room_id);
-  } else if (room_id.empty()) {
-    LOG(WARNING) << "Neither room name nor room ID specified, exiting";
```

Comment:
This seems to work with CI, but I don't quite follow *how* &ndash; with this removed, I've have expected the `run_examples` script to hang on this example.

Reply:
Ah, I think https://github.com/project-oak/oak/pull/433 actually broke this expectation, since Rust modules are not called `module` any more. I will follow up with a separate PR (see https://github.com/project-oak/oak/issues/458), though I am not sure what to do with this example. Maybe add a `--dry-run` flag?